api/routes.py

```diff
@@ -66,3 +66,24 @@ async def read_price_data(price_data_id: int, db: Session = Depends(get_db)):
         "time_start": price_data.time_start,
         "time_end": price_data.time_end
     }
+
+# GET endpoint: Fetch data by Zone
+@router.get("/price-data/{price_data_zone}")
+async def read_price_data_zone(price_data_zone: int, db: Session = Depends(get_db)):
```

Comment:
*I think it should be price_data_zone: str

Reply:
changed to str